clevercsv/console/commands/standardize.py

```diff
@@ -178,23 +191,24 @@ def handle_path(
             return 1
 
         if self.args.in_place:
-            return self._in_place(path, dialect, encoding)
+            return self._in_place(path, dialect, encoding, target_encoding)
         elif output is None:
-            return self._to_stdout(path, dialect, encoding)
-        return self._to_file(path, output, dialect, encoding)
+            return self._to_stdout(path, dialect, encoding, target_encoding)
+        return self._to_file(path, output, dialect, encoding, target_encoding)
 
     def _write_transposed(
         self,
         path: StrPath,
         stream: SupportsWrite[str],
         dialect: SimpleDialect,
         encoding: Optional[str],
+        target_encoding: Optional[str]
     ) -> None:
         with open(path, "r", newline="", encoding=encoding) as fp:
             read = reader(fp, dialect=dialect)
             rows = list(read)
         rows = list(map(list, zip(*rows)))
-        write = writer(stream, dialect="excel")
+        write = writer(stream, dialect="excel", encoding=target_encoding or encoding)
```

Comment:
This logic (`encoding=target_encoding or encoding`) is repeated in several places, which will make it more difficult to maintain if it ever needs updating. I suggest to do this on line 128 when you initialize `target_encoding`, so it's only done once.

Reply:
Done in https://github.com/alan-turing-institute/CleverCSV/commit/a43a77768bef1faff0eb4b6055f6d3c4acff4745
In line 186 instead of 128, because I don't think it relevant to have multiples target-encoding, is it ?

Reply:
Agreed, only one target encoding should suffice